src/SDKs/HDInsight/Management/Management.HDInsight/generated.cmd

```diff
@@ -4,4 +4,4 @@
 ::
 
 @echo off
-call %~dp0..\..\..\..\tools\generate.cmd hdinsight/resource-manager %*
+call %~dp0..\..\..\..\..\tools\generate.cmd hdinsight/resource-manager %*
```

Comment:
you no longer need this file.
Rather you should be using the .PS1 file that you see in other SDKs.
Model your .ps1 by looking at any other SDK directory.

Reply:
Hi @shahabhijeet , I will rename the .csproj file. But what about the namespace? We want to keep "Microsoft.Azure.Management.HDInsight" because it is from original [Hyak based Job SDK](https://github.com/Azure/azure-sdk-for-net/blob/master/src/ResourceManagement/HDInsightJob/HDInsightJob/HDInsightJob.csproj#L9-L10). Does it make sense to you? Could you please give us some suggestions on this topic?

Reply:
I will create .ps1 in the next iteration.

Reply:
@idear1203 it is important you generate the SDK using the ps1, by doing so you will be able to generate SDK with the proper meta data.
Also renaming of the files is important to be done now. Unless you have a reason why it should be done next time and not now?

Reply:
Oh. I mean I will do now in the new commit. Just want to check if you think  "Microsoft.Azure.HDInsight.Job" is a good namespace?

Reply:
@idear1203 is Microsoft.Azure.HDInsight.Job a resource provider or is it Microsoft.Azure HDInsight?

Reply:
No, it is not a resource provider, it is a wrapper of open source [Apache Templeton API](https://cwiki.apache.org/confluence/display/Hive/WebHCat+UsingWebHCat). We are moving from Hydra to Swagger. The codebase for the hydra spec that generates the Jobs SDK is [here](https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fhydra-specs-pr%2Ftree%2Fmaster%2FHDInsight2%2FMicrosoft.Azure.Management.HDInsight.Job.Specification&data=02%7C01%7CDongwei.Wang%40microsoft.com%7C53ce95cbabe143dfa92308d6345c6e8a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636753968976913095&sdata=i9A%2FF%2FdTv6W35u2YCpITthDucA7Ap6wlzfbC3BsZ%2FBY%3D&reserved=0) and the codebase for the generated Jobs SDK is [here](https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-sdk-for-net%2Ftree%2Fmaster%2Fsrc%2FResourceManagement%2FHDInsightJob%2FHDInsightJob&data=02%7C01%7CDongwei.Wang%40microsoft.com%7C53ce95cbabe143dfa92308d6345c6e8a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636753968976913095&sdata=ZqTPGEHURqL4THNqoeH70BtBoyxZga2sYZmVzx7mwIg%3D&reserved=0).

We would like to name the package as "Microsoft.Azure.HDInsight.Job" rather than "Microsoft.Azure HDInsight", since all this package is related to job management and in the old Hydra-based SDK, the namespace is "Microsoft.Azure.Management.HDInsight.Job". We thought it would be nice if we could keep "Microsoft.Azure.Management.HDInsight.Job". But we think it makes sense if we need to use "Microsoft.Azure.HDInsight.Job". What do you think?